app/assets/javascripts/discourse/templates/list/posters-column.raw.hbs

```diff
@@ -1,5 +1,5 @@
 <td class='posters'>
 {{#each posters as |poster|}}
-<a href="{{poster.user.path}}" data-user-card="{{poster.user.username}}" class="{{poster.extraClasses}}">{{avatar poster avatarTemplatePath="user.avatar_template" usernamePath="user.username" imageSize="small"}}</a>
+<a href="{{poster.user.path}}" data-user-card="{{poster.user.username}}" class="{{poster.extraClasses}}">{{avatar poster avatarTemplatePath="user.avatar_template" usernamePath="user.username" namePath="user.name" imageSize="small"}}</a>
```

Comment:
It'll be good if we can add a QUnit acceptance test for this.

Reply:
Done.